### ISSUE TITLE
lorenztian typo

### ISSUE THREAD

**Issue by Kawue** [mobiusklein/ms_peak_picker#5]
Thanks for this nice piece of code.
There is a small typo in your peak_picker class. Its consistently wrong, so it does not matter but your documentations states it as "lorentzian". Would be nice to adjust either the code or the comments.

**Comment by mobiusklein** [comment]
Thank you for catching that. 

I'll fix the spelling in the code itself, but the old misspelling will still work when selecting fit type in `pick_peaks`.

I've found the Lorentzian fit didn't find the centroid very well when dealing with clipped peaks, which happen fairly frequently when operating the mass spectrometer for throughput.

**Comment by mobiusklein** [comment]
I've made the name changes. Both "lorentzian" and "lorenztian" will work with `fit_type`. I noticed I've disabled the C implementation of the Lorentzian least squares, and it appears to be a long standing setting. I've re-enabled them after some manual testing on some "clean" data sets. If I remember correctly, there were intermittent issues with the full width at half max calculation it required early in this library's development, and I couldn't demonstrate it was objectively better than the simpler quadratic fit.

Theoretically, an FTICR's peak shape should be closer to Lorentzian.

![image](https://user-images.githubusercontent.com/1426041/53279241-bd692380-36dc-11e9-88d5-0b13547969ad.png)

![image](https://user-images.githubusercontent.com/1426041/53279252-db368880-36dc-11e9-90c4-1f16b443226b.png)

But my experience has been that Orbitrap peaks tend to be more Gaussian.

![image](https://user-images.githubusercontent.com/1426041/53279417-2dc47480-36de-11e9-9104-4afbfe8cd153.png)
